secp256k1-sys/depend/secp256k1/src/util.h

```diff
@@ -145,11 +145,9 @@ static const rustsecp256k1_v0_9_0_callback default_error_callback = {
 #endif
 
 static SECP256K1_INLINE void *checked_malloc(const rustsecp256k1_v0_9_0_callback* cb, size_t size) {
```

Comment:
Maybe delete it completely to ensure it's not called by accident?

Reply:
Then I would need to delete all the functions that call it, and so on, which is much more invasive.

Reply:
Sucks. Also I wonder how does it interact with the system library. Will anything break if it has different code?

Reply:
No. It's not an exposed function, it's only used internally.